docs/content/docs/features/primary-key-table.md

```diff
@@ -158,12 +154,21 @@ Full compaction changelog producer can produce complete changelog for any type o
 
 {{< /hint >}}
 
-## Changelog Tables without Primary Keys
+## Sequence Field
 
-Changelog tables can also be used without primary keys. Users can only insert or delete a whole record from the table. No update is supported.
+By default, the primary key table determines the order according to the input order. However, in distributed computing,
```

Comment:
By default, primary key tables determine the latest record according to the input order.

Reply:
Change to: `By default, the primary key table determines the merge order according to the input order`.